db/migrate/20230823124431_create_tasks.rb

```diff
@@ -0,0 +1,21 @@
+class CreateTasks < ActiveRecord::Migration[7.0]
+
+	def up
+		create_table :tasks, id: false do |t|
+			t.integer :task_id, primary_key: true
+			t.integer :user_id 
```

Comment:
is this the foreign key ? If so, then please define it as follows
```suggestion
			t.integer :user_id, foreign_key: true
```